pkg/admin/transport/handler_user_export_get.go

```diff
@@ -21,9 +22,10 @@ type UserExportGetProducer interface {
 }
 
 type UserExportGetHandler struct {
-	AppID       config.AppID
-	JSON        JSONResponseWriter
-	UserExports UserExportGetProducer
+	AppID        config.AppID
+	JSON         JSONResponseWriter
+	UserExports  UserExportGetProducer
+	CloudStorage cloudstorage.Storage
```

Comment:
Introduce a new interface for dependency injection. This is our convention.

```
type UserExportGetHandlerCloudStorage interface {
  PresignGetObject(key string, expire time.Duration) (*url.URL, error)
}
```

Of course, you need to set up wire to bind it.

Reply:
added new interface for this purpose